### PR TITLE
added commonly requested configs to basic slurp.

### PR DESCRIPTION
**Pull request by ryanbrinn**
It doesn't look like the existing templates allow a predefined field name to be passed in.

### CODE REVIEW

File: slurp_all_MODS_to_solr.xslt [hunk 1]
```diff
@@ -355,4 +357,68 @@
...
+      <xsl:with-param name="content" select="normalize-space()"/>
+    </xsl:call-template>
+  </xsl:template>
+  <xsl:template match="mods:mods/mods:genre" mode="slurp_all">
+    <xsl:call-template name="mods_all_fork">
+      <xsl:with-param name="field_name" select="'genre_all'"/>
+      <xsl:with-param name="content" select="normalize-space()"/>
+    </xsl:call-template>
+  </xsl:template>
+  <xsl:template match="mods:mods/mods:name[@type or not(@type)][mods:role/mods:roleTerm[@authority= 'marcrelator' and @type = 'text']][. = 'contributor']/mods:namePart" mode="slurp_all">
```
Comment (qadan) [inline slurp_all_MODS_to_solr.xslt]: `@type or not(@type)` seems redundant?

**Comment by qadan** [comment]
Just a general note: these additions don't seem like they belong in the general "slurp", and referring to them in a "slurp_all" mode also seems misleading ... the point of the slurp is that singling out nodes should only be used to fork the field name construction; everything else is all automated.

Could we move this to a separate XSLT referenced by inclusion, much like we do for our current customizations? This'd give us the option to add more in the future while maintaining the separation between the "slurp" and the "specific nodes" field types.

**Comment by ryanbrinn** [comment]
I'm not sure what to do with this one. This seems like a conversation we should have had before code review, no? @jordandukart can you comment on this?

**Comment by ryanbrinn** [comment]
nvm I understand this. I had the dumb

**Comment by qadan** [comment]
Two just general comments and then I think that's it from me:

1. Can we make the field names match our regular slurp concatenation method, e.g., `mods_physicalDescription_form_all_ms` instead of `mods_form_all_ms`, or `mods_name_contributor_namePart_all_ms` instead of `mods_contributor_all_ms`; mods_ fields essentially represent an API at this point, and given this change is intended to alleviate confusion over the purpose of different fields, maintaining consistency with field name construction is probably important.

2. Not reeeeeally important, but you could cut out 8 instances of `<xsl:with-param name="content" select="normalize-space()"/>` if you gave `mods_all_suffix`'s `content` field a default ;) I'm not deeply concerned with playing XSLT golf though.

**Pull request merged**